dashing/package.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>cyclonedds</name>
+  <version>0.1.0</version>
+  <description>Performant and robust open-source DDS implementation.</description>
+  <maintainer email="logans@cottsay.net">Scott K Logan</maintainer>
+  <license>EPL-2.0</license>
```

Comment:
This is how Fedora describes `Eclipse Public License v2.0`. Should I spell it out like that here, or is this sufficient?

Reply:
I don't mind. I am fine with how it is.

Reply:
EPL-2.0 is the correct SPDX license identifier. https://spdx.org/licenses/EPL-2.0.html